README.md

```diff
@@ -2,19 +2,34 @@
 SOEN 341 Photo-sharing Social Network Project - Winter 2020
 
 ## Objective
-Develop a photo-sharing social network platform.
+Develop a photo-sharing social network platform, with a goal to create a simplified version of Instagram.
+
+## Description
+The end-product of our project shall contain the core features below complemented with unique and new features determined by the team.
+This is a project developed in the scope of the SOEN 341 course at Concordia university. The web app will allow its users to share their photos with friends and to message each other.
+
+## Core Features
+1. The software shall allow the user to post a picture.
+2. The software shall allow users to follow others.
+   - 2.1 All followers shall receive a notification of a new post.
+3. The software shall support user comments on all posts.
 
 ## Project
 Project description and key features
 
 ## Team Members
-- Jean-Philippe Durand
-- Nathan Ziri
-- Pierre-Alexis Barras
-- Jasen Ratnam
-- Alya Naseer
+- Jean-Philippe Durand (Git Username: Aeroponz)
+- Nathan Ziri (Git Username: NathanZiri)
+- Pierre-Alexis Barras(Git Username: Pyxsys)
+- Jasen Ratnam (Git Username: JasenRatnam)
+- Alya Naseer (Git Username: AlyaNaseer)
 
 ## Tools and Technologies
-- Tool1
-- Tool2
-- Tool3
+- Languages:
+  - Java
+  - PHP
+  - HTML
+  - CSS
+- Technology:
```

Comment:
Add "Apache web server" (a new issue updating this may be required in the future)